__tests__/app.test.js

```diff
@@ -340,3 +340,34 @@ describe("PATCH /api/articles/article_id:", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/comment_id:", () => {
+  test("204: Deletes the comment from the database", () => {
+    return request(app)
+      .delete("/api/comments/3")
+      .expect(204)
+      .then(() => {
+        return db
+          .query(`SELECT * FROM comments WHERE comment_id = 3`)
+          .then(({ rows }) => {
+            expect(rows.length).toBe(0);
+          });
+      });
```

Comment:
Nice way of checking whether the comment is still there, to be honest the 204 status alone is enough as that should tell you that it has actually deleted even though nothing is sent over. But I like the idea!